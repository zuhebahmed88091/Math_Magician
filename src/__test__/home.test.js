import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/home';

describe('Testing home page', () => {
  it('Home page shows', () => {
    const homeShows = render(<BrowserRouter><Home /></BrowserRouter>);
    expect(homeShows).toMatchSnapshot();
  });
});
