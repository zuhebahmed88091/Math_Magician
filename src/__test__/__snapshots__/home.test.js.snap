// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing home page Home page shows 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <article
        class="home"
      >
        <h1>
          Welcome to my page
        </h1>
        <p>
          For all math lovers, there is a website called Math Magicians. It is a Single Page App (SPA) that enables users to read a random quote about arithmetic and perform basic calculations. Enjoy
        </p>
        <footer
          class="footer"
        >
          © Zuheb Ahmed
        </footer>
      </article>
    </div>
  </body>,
  "container": <div>
    <article
      class="home"
    >
      <h1>
        Welcome to my page
      </h1>
      <p>
        For all math lovers, there is a website called Math Magicians. It is a Single Page App (SPA) that enables users to read a random quote about arithmetic and perform basic calculations. Enjoy
      </p>
      <footer
        class="footer"
      >
        © Zuheb Ahmed
      </footer>
    </article>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
